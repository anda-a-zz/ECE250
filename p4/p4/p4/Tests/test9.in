n 4
i 1;2;12
i 1;0;10
edge_count
degree 1
degree 0
d 1;0
edge_count
i 2;3;15
i 3;0;20
edge_count
degree 3
mst
