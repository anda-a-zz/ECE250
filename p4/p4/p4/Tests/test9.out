success
success
success
edge count is 2
degree of 1 is 2
degree of 0 is 1
success
edge count is 1
success
success
edge count is 3
degree of 3 is 2
mst 47
